src/main/java/org/team1540/robot2023/commands/arm/ExtensionPID.java

```diff
@@ -0,0 +1,24 @@
+package org.team1540.robot2023.commands.arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ExtensionPID extends CommandBase {
```

Comment:
Maybe something like AutoExtendCommand? Based on the current name, I would assume this is a subclass of the PIDController class, not a command